gensrc/script/doris_builtins_functions.py

```diff
@@ -533,6 +533,8 @@
             '15FunctionContextERKNS1_9StringValERKNS1_6IntValES6_'],
     [['length'], 'INT', ['VARCHAR'],
             '_ZN5doris15StringFunctions6lengthEPN9doris_udf15FunctionContextERKNS1_9StringValE'],
+    [['char_length'], 'INT', ['VARCHAR'],
```

Comment:
Better to give a synonym CHARACTER_LENGTH, which is also supported in MySQL

Reply:
@imay done

Reply:
```suggestion
    [['char_length', 'character_length'], 'INT', ['VARCHAR'],
```
Prefer above type

Reply:
Maybe you forget to add this file